webapp/src/main/webapp/src/app/groups/delete-group.modal.html

```diff
@@ -0,0 +1,23 @@
+<div *ngIf="group">
+  <div class="modal-header">
+    <h4 class="modal-title pull-left">{{ 'labels.groups.delete.title' | translate:group }}</h4>
+    <button type="button" class="close pull-right" title="{{'prompt.close' | translate}}" (click)="cancel()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body" [innerHtml]="'labels.groups.delete.content-html' | translate">
+  </div>
+  <div *ngIf="unableToDelete" class="alert alert-danger">
+    {{ 'labels.groups.delete.error' | translate }}
+  </div>
+  <div class="modal-footer">
+    <span>
+      <button type="button" class="btn btn-default" (click)="cancel()">
+        {{ (unableToDelete? 'prompt.close' : 'labels.groups.delete.button-cancel') | translate }}
+      </button>
+      <button type="button" *ngIf="!unableToDelete" class="btn btn-primary" (click)="save()">
```

Comment:
I like how this looks, thanks.

@aaronsleeper thoughts??

![image](https://user-images.githubusercontent.com/5771116/29378872-8b187fce-8275-11e7-8733-79d516e96334.png)